styleguide/derek/assets/index.jade

```diff
@@ -14,9 +14,15 @@ block page
           li
             a(href="derek/assets/typography") Typography &amp; Translations
           li
+            a(href="derek/assets/swatches") Colors &amp; Swatches
+          li
             a(href="derek/assets/icon-font") Icon Font
           li
             a(href="derek/assets/photography") Photography &amp; Banners
+          li
+            a(href="derek/assets/tables") Tables
+          li
+            a(href="derek/assets/tables") Forms
```

Comment:
Incorrect link here. Also if we're gonna link to the form styles, we probably should remove them from the asset page itself:
https://github.com/amberRrucker/zoolander/blob/c2f20da020cb28fa9661e9de89eb6a3a84b72428/styleguide/derek/assets/index.jade#L79